mlrun/config.py

```diff
@@ -547,6 +547,10 @@
                 "max_size": 10000,
             },
         },
+        "pipelines": {
```

Comment:
move it under `"projects": { .. "summaries": {`

`projects.summaries.list_pipelines_time_period_in_days`